contents/spotlight/startup-puzzle.md

```diff
@@ -0,0 +1,79 @@
+
+“More often than not, founders aren’t signing up to Puzzle because they’re excited to do their accounting,” says Scott. “They sign up because they know they need to be tax compliant, gain a deeper understanding of their company’s finances, and be ready for things like investor reporting. While the needs of founders often change or mature at each stage of their journey, what’s common is that they care about compliance, efficiency, and accuracy.”
+
+All of this is reflected in Puzzle’s online messaging, with language which directly addresses founders ahead of accountants and emphasizes how the product integrates simply with other tools which founders already use. 
+
+> **Putting it into practice:** [We advise having an approximate structure for user interviews](/blog/how-to-work-out-what-users-need), but leave room to flex. If users go off on a tangent during an interview that could give you vital [insights to their actual feelings](/handbook/product/user-feedback)!
+
+## 3. Build pricing around user segments
+
+Next, you can delve into the quantitative data by looking at how users can be segmented into different groups based on criteria such as size or industry. Puzzle used this information to structure its pricing model, for example. 
+
+“On [our pricing page](https://puzzle.io/pricing) we make this pretty clear,” says Scott. “We have a plan for formation-stage founders, a plan for startups who have raised a little funding, and an automation plan for companies which are a little more mature and prioritize efficiency, accuracy, and not waiting weeks to close their books."
+
+Each plan also reflects the pain points a user would deal with at that stage too and addresses it with a product feature. Monthly compliance checklists in the free plan, for example, help formation-stage founders get started quickly — but give way in the Automation plan to AI and streamlined workflows.
+
+> **Putting it into practice:** If you aren’t using tiered pricing plans then it can still be helpful to think of your broader [pricing principles](/handbook/engineering/feature-pricing). Ask yourself: how do these principles work for your different user segments? 
+
+## 4. Reinforce your persona to internal teams
+
+As you grow, it’s important not to drift from targeting your persona or to let teams’ understanding of it become corrupted. You need to constantly remind everyone in your company who you are building for and how your product meets their needs — otherwise you risk diluting your offering. 
+
+“When I joined Puzzle I was very happy to hear people talking about our ICP,” says Scott. “While there were still debates about some of the finer details and who some of the buyer personas may be, it was clear that our ICP was our guiding light.”
```

Comment:
Maybe ICP vs persona needs to be briefly explained at the start.

Reply:
```suggestion
“When I joined Puzzle I was very happy to hear people talking about our target persona,” says Scott. “While there were still debates about some of the finer details and who some of the buyer personas may be, it was clear that our target persona was our guiding light.”
```